assignments/sa/starterpack/index.md

```diff
@@ -141,6 +141,8 @@ Let's set up a simple `index.html` to act as our main html page.
     <title>starter pack</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+		<!-- Adding a favicon is optional, but will help your users differentiate browser tabs -->
+		<link rel="shortcut icon" href="favicon.png" type="image/x-icon" />
```

Comment:
great idea!  wouldn't the HtmlWebpackPlugin insert this itself if it wasn't here?   also does the favicon in this case need to be any particular format?  for instance would using https://github.com/jantimon/favicons-webpack-plugin help if your png happens to be too large etc?

Reply:
Thanks for getting back to me!

I think what you proposed is certainly more robust. I think it is solving a different problem.

In this PR, I was specifically thinking about the favicon showing up while developing at localhost:8080 (for example). When I completed the assignment, I noticed that the favicon wasn't showing up on localhost (even though it worked on the deployed site).

Reply:
got it!  looks like the webpack line does both without needing this link, will edit and merge.  thanks!